activity/core.go

```diff
@@ -107,6 +107,8 @@ func Warnf(ctx context.Context, msg string, a ...interface{}) {
 // SubmitMethodInvocation submits a method invocation event to Google Analytics.
 // It then records the invocation to the journal identified by the ID at `activity.JournalKey`
 // in the provided context.
+//
+// Note that entryType will typically be the type ID.
```

Comment:
I want to keep entryType as-is here in case we do decide that reporting the entry type is useful.